apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoBar.tsx

```diff
@@ -190,8 +192,11 @@ const CourseInfoBar = (props: CourseInfoBarProps) => {
                     const currentTarget = event.currentTarget;
                     void togglePopover(currentTarget);
                 }}
+                style={{ minWidth: 200 }}
             >
-                {`${deptCode} ${courseNumber} | ${courseTitle}`}
+                <span
+                    style={{ whiteSpace: 'nowrap', overflow: 'hidden', textAlign: 'left' }}
+                >{`${deptCode} ${courseNumber} | ${courseTitle}`}</span>
```

Comment:
nit (prettier formatting):

```suggestion
                <span style={{ whiteSpace: 'nowrap', overflow: 'hidden', textAlign: 'left' }}>
                    {`${deptCode} ${courseNumber} | ${courseTitle}`}
                </span>
```